tests/topotests/bgp_srv6l3vpn_over_ipv6/r1/bgpd.conf

```diff
@@ -0,0 +1,52 @@
+frr defaults traditional
+!
+hostname r1
+password zebra
+!
+log stdout notifications
+log monitor notifications
+log commands
+!
+router bgp 65001
+ bgp router-id 1.1.1.1
```

Comment:
Can you replace this with http://docs.frrouting.org/projects/dev-guide/en/latest/topotests.html (^f+`RFC 5737`)